astropy/units/tests/test_quantity_typing.py

```diff
@@ -0,0 +1,108 @@
+# coding: utf-8
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Test the Quantity class and related."""
+
+import sys
+import typing as T
+
+import pytest
+
+from astropy import units as u
+
+""" The Quantity class will represent a number + unit + uncertainty """
+
+
+class TestQuantityTyping:
+    """Test Quantity Typing Annotations.
+
+    .. todo::
+
+        - switch tests to ``get_args``, ``get_origin`` when min python 3.8+
+
+    """
+
+    def test_quantity_typing(self):
```

Comment:
Here, you do have nice methods! I'd split off a few more (essentially any time you have a 2-line comment!)

Reply:
Thanks. Done!